behat/tests/bootstrap/v2/common/IShouldBeOnTrait.php

```diff
@@ -63,11 +63,51 @@ public function iAmOnContactsAddAnotherPage()
         $this->iAmOnPage('/report\/.*\/contacts\/add_another$/');
     }
 
+    /**
+     * @Then I should be on the financial decision actions page
+     */
+    public function iAmOnActionsPage1()
+    {
+        $this->iAmOnPage('/report\/.*\/actions\/step\/1$/');
+    }
+
+    /**
+     * @Then I should be on the concerns actions page
+     */
+    public function iAmOnActionsPage2()
+    {
+        $this->iAmOnPage('/report\/.*\/actions\/step\/2$/');
+    }
+
+    /**
+     * @Then I should be on the actions report last step page
+     */
+    public function iAmOnActionsLastStepPage()
+    {
+        $this->iAmOnPage('/report\/.*\/actions\/summary\?from\=last\-step$/');
+    }
+
+    /**
+     * @Then I should be on the actions report summary page
+     */
+    public function iAmOnActionsSummaryPage()
+    {
+        $this->iAmOnPage('/report\/.*\/actions\/summary/');
+    }
+
+    /**
+     * @Then I should be on the Lay main overview page
```

Comment:
Maybe:

`Then I should be on the Lay homepage`?

Reply:
Hmm yeah that sounds nicer